allennlp_models/pretrained.py

```diff
@@ -1,194 +1,34 @@
-    """
-    Fine Grained Named Entity Recognition with the transformer
-    """
-    predictor = _load_predictor(
-        "https://storage.googleapis.com/allennlp-public-models/fgner_transformer.2020-07-14.tar.gz"
-    )
-    return predictor
-
-
-def span_based_constituency_parsing_with_elmo_joshi_2018() -> ConstituencyParserPredictor:
-    """
-    Constituency Parsing
-
-    Based on `Minimal Span Based Constituency Parser (Stern et al, 2017) <https://www.semanticscholar.org/paper/A-Minimal-Span-Based-Neural-Constituency-Parser-Stern-Andreas/593e4e749bd2dbcaf8dc25298d830b41d435e435>`_ but with ELMo embeddings
-    """
-    predictor = _load_predictor(
-        "https://storage.googleapis.com/allennlp-public-models/elmo-constituency-parser-2020.02.10.tar.gz"
-    )
-    return predictor
-
-
-def biaffine_parser_universal_dependencies_todzat_2017() -> BiaffineDependencyParserPredictor:
-    """
-    Biaffine Dependency Parser (Stanford Dependencies)
-
-    Based on `Dozat and Manning, 2017 <https://arxiv.org/pdf/1611.01734.pdf>`_
-    """
-    predictor = _load_predictor(
-        "https://storage.googleapis.com/allennlp-public-models/biaffine-dependency-parser-ptb-2020.04.06.tar.gz"
-    )
-    return predictor
-
-
-def esim_nli_with_elmo_chen_2017() -> DecomposableAttentionPredictor:
+    Returns a Dict of model cards of all available pretrained models.
     """
-    ESIM
-
-    Based on `Enhanced LSTM for Natural Language Inference <https://arxiv.org/pdf/1609.06038.pdf>`_ and uses ELMo
-    """
-    predictor = _load_predictor(
-        "https://storage.googleapis.com/allennlp-public-models/decomposable-attention-elmo-2020.04.09.tar.gz"
-    )
-    return predictor
-
-
-def glove_sst() -> TextClassifierPredictor:
-    return _load_predictor(
-        "https://storage.googleapis.com/allennlp-public-models/basic_stanford_sentiment_treebank-2020.06.09.tar.gz"
-    )
-
-
-def roberta_sst() -> TextClassifierPredictor:
-    return _load_predictor(
-        "https://storage.googleapis.com/allennlp-public-models/sst-roberta-large-2020.06.08.tar.gz"
-    )
-
-
-def roberta_mnli() -> DecomposableAttentionPredictor:
-    return _load_predictor(
-        "https://storage.googleapis.com/allennlp-public-models/mnli_roberta-2020.06.09.tar.gz",
-        "textual-entailment",
-    )
-
-
-def roberta_snli() -> DecomposableAttentionPredictor:
-    return _load_predictor(
-        "https://storage.googleapis.com/allennlp-public-models/snli_roberta-2020.06.09.tar.gz",
-        "textual-entailment",
-    )
 
+    # These imports are included so that the model cards can be filled with default information
+    # obtained from the registered model classes.
 
-def roberta_piqa() -> TransformerMCPredictor:
-    return _load_predictor(
-        "https://storage.googleapis.com/allennlp-public-models/piqa.2020-07-08.tar.gz"
+    pretrained_models = {}
+    model_card_paths = os.path.join(
+        os.path.dirname(os.path.realpath(__file__)), "modelcards/*.json"
```

Comment:
If you want to do it this way, you have to make sure those json files are included in the wheel that `setup.py` builds.